src/styles/Home.module.css

```diff
@@ -72,6 +74,41 @@
   background-color: #3023b8;
 }
 
+/* CM DASHBOARD */
+.cmContainer {
```

Comment:
This should ideally be in its own module file

Reply:
I agree, I just put it here for now since this was a minimal styling attempt - but I'll make sure to do so in a future PR with more styling changes! @amantri